.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+indent_style = space
```

Comment:
```gofmt``` uses tabs, for indentation. See:

https://golang.org/src/cmd/gofmt/doc.go

for details.

I think you can just add a section for makefiles, as in most cases, golang integration into IDEs will take care of linting, formatting, etc. This is useful for people who prefer ```vim``` though. For ```golang``` however it would be more useful to add a make target that does:

```
gofmt -s -w -l .
```

This will list all files who's formatting differs from gofmt's. Editors should be set to automatically run ```gofmt -s -w``` on file save.